### ISSUE TITLE
[Keccak] Tests with corner cases.

### ISSUE THREAD

**Issue by dannywillems** [o1-labs/proof-systems#1327]
Would have been nice to have some tests with corner cases.
What about messages of 1088 bits? The documentation does not mention it. Reading the code makes me feel it adds a full block. But tests can confirm this.

_Originally posted by @dannywillems in https://github.com/o1-labs/proof-systems/pull/1262#discussion_r1381757140_

**Comment by querolita** [comment]
Yeah, it adds one full block just for padding. Will make this clearer in doc and tests.
